cmd/grafana-agent/entrypoint.go

```diff
@@ -352,6 +352,22 @@ func (ep *Entrypoint) TriggerReload() bool {
 	return true
 }
 
+// Triggers a reload of the config on each tick of the ticker until the context
```

Comment:
```suggestion
// pollConfig triggers a reload of the config on each tick of the ticker until the context
```
Comments for function in Go always start with the name of the function. This is needed for generating the godoc, in case of exported functions, but for consistency it's a good practice to use same format for unexported functions.

Reply:
Ah good to know, I'll go through all of the added functions and update their comments!